src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1

```diff
@@ -103,6 +103,22 @@ InModuleScope Arcus.Scripting.Sql {
             } catch {
                 # We don't care if an exception is thrown; we just want to 'activate' the Azure SQL database.
             }
+
+            $tableNames = @()
+            try {
+                $tableNames = Run-AzSqlQuery $params "SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES"
+            } catch {
+                Write-Warning "Could not select any SQL tables to drop during test setup"
+            }
+            
+            foreach ($tableName in $tableNames.ItemArray) {
+                try {
+                    Write-Verbose "Drop table $tableName in SQL database"
+                    Drop-AzSqlDatabaseTable $params $tableName
```

Comment:
This is not taking into account specific schemas, it defaults back to `dbo` due to `function global:Drop-AzSqlDatabaseTable ($params, $databaseTable, $schema = "dbo")` on line 19.

Not sure if we have any tables that are in a specific schema other than `dbo` that need to be dropped?

Reply:
Hmm, good catch. The information_schema query will get all our tables, but doesn't take into account schemes, indeed. Will need to add that.

Reply:
Added in https://github.com/arcus-azure/arcus.scripting/pull/263/commits/f1bd166a4bd475a944300c7c18cb9e02654d40b6